src/lib/Components/Lists/SavedFairItemRow.tsx

```diff
@@ -0,0 +1,179 @@
+import React from "react"
+import styled from "styled-components/native"
+
+import { Flex } from "@artsy/palette"
+import { SavedFairItemRowMutation } from "__generated__/SavedFairItemRowMutation.graphql"
+import OpaqueImageView from "lib/Components/OpaqueImageView"
+import { Colors } from "lib/data/colors"
+import fonts from "lib/data/fonts"
+import Switchboard from "lib/NativeModules/SwitchBoard"
+import moment from "moment"
+import { TouchableWithoutFeedback } from "react-native"
+import { commitMutation, graphql, RelayProp } from "react-relay"
+
+interface State {
+  isSaved: boolean
+}
+
+const Container = styled.View`
+  margin: 9px 20px 0;
+  height: 95px;
+`
+
+const Content = styled.View`
+  display: flex;
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+`
+
+const SaveLabel = styled.Text`
+  font-family: ${fonts["unica77ll-regular"]};
```

Comment:
we shouldn't use fonts directly here, that should come from `palette`. You can extend our base font components the following way.

```typescript
styled(Sans).attrs({
  weight: "medium",
  color: "black60"
})`
  ....
  text-align: right;
`
```

but in this case you actually could use `<Sans weight="medium" />` directly